docs/QuickStart.md

```diff
@@ -1,64 +1,101 @@
+  
+- **Aspect**  
+  A characteristic of a subject. An aspect can be anything that is measurable, such as average page times, wait times for a train, and even the weather. Anything!
+  
+- **Sample**  
+  The value of an aspect for a particular subject at a specified point in time. For example, a snapshot of subject status and health.
+  
+- **Lens**  
+  A visualization of the underlying Refocus data. Each lens provides a unique layout to represent the subject, child subjects, and samples.  
+  
+  With the Lens Developer Kit (**coming soon**), you can create your own lens to meet your specific visualization needs. When you install your lens into Refocus, it will also be available to other users.
+  
+- **Perspective**  
+  A filtered view of a subset of data using a selected lens. You can save a perspective and share it with other Refocus users.
+  
+## Refocus Interface
+
+Refocus displays real-time system status in a browser window based on the selected lens. The following lenses are available for Refocus.  
+- **MultiTable**  
+  A set of tables with the various subject components arranged like a menu board. You can view system status at a glance and expand the components for combos and sides.  
+  
+- **Branch**  
+  An expandable set of system components, organized by the hierarchy each lives in. Click a branch to explore the component neighborhood.  
+  
+- **Tree**  
+  An expandable display of system components, organized in a root, trunk, and branch hierarchy. Go out on a limb to view the status of individual components.  
+  
+You can develop your own lenses with the [lens developer kit](https://github.com/salesforce/refocus/blob/master/docs/QuickStart.md#create-your-own-lens) (LDK).
+
+## Getting Started with Refocus  
+You have two ways to deploy Refocus. Select the one that best meets your needs.
 
-Before you go any further, now might be a good time to [get familiar with some of the terms we use in Refocus](./Glossary.md).
+- I want [one-click Heroku deployment](./QuickStartHeroku.md)
+- I want to [download and build and deploy locally](./QuickStartLocal.md)
 
-_OK, enough chit chat, let's get started already!_
+Refocus requires that both Redis and PostgreSQL are running. After Refocus is deployed, create an account and sign in.
 
-# Quick Start
+*OK, I've got Refocus deployed, and I'm signed in. Now what?*  
 
-There are two options to set up Refocus. Select the one that best meets your needs:
-- I want [one-click Heroku deployment](./QuickStartHeroku.md)
-- I want to [download and build and deploy locally](./QuickStartLocal.md)
+Think about how you want to structure your visualization based on the Refocus hierarchy. The relationship between subjects, aspects, and samples looks like this:
 
-Ensure that your deployment, either on Heroku or locally, has both Redis and PostgreSQL running. Once deployed, create an account and sign in.
+![Subject Hierarchy](
```

Comment:
@iamigo @meynet I need permission to upload files to the repo folders so I can add images to this doc. This is a placeholder link to Jee's hierarchy graphic.